### PR TITLE
Remove unneeded method

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/Features/CSharp/Portable/Completion/KeywordRecommenders/BaseKeywordRecommender.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ private bool IsConstructorInitializerContext(int position, CSharpSyntaxContext c
             if (token.Kind() == SyntaxKind.ColonToken &&
                 token.Parent is ConstructorInitializerSyntax &&
                 token.Parent.IsParentKind(SyntaxKind.ConstructorDeclaration) &&
-                token.Parent.GetParent().IsParentKind(SyntaxKind.ClassDeclaration))
+                token.Parent.Parent.IsParentKind(SyntaxKind.ClassDeclaration))
```
Comment (CyrusNajmabadi) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/BaseKeywordRecommender.cs]: i didn't just convert all calls to `GetParent()` with `?.Parent`.  If the preceding checks prove that the left is not null, we can just use `.Parent`.

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide @mavasani .  Simple PR.

File: src/Workspaces/Core/Portable/Shared/Extensions/SyntaxNodeExtensions.cs [hunk 2]
```diff
@@ -762,8 +762,7 @@ public static SyntaxNode GetParent(this SyntaxNode node, bool ascendOutOfTrivia)
             var parent = node.Parent;
             if (parent == null && ascendOutOfTrivia)
             {
-                var structuredTrivia = node as IStructuredTriviaSyntax;
-                if (structuredTrivia != null)
```
Comment (sharwell) [inline src/Workspaces/Core/Portable/Shared/Extensions/SyntaxNodeExtensions.cs]: 📝  This file should not be part of this change.

Comment (CyrusNajmabadi) [reply src/Workspaces/Core/Portable/Shared/Extensions/SyntaxNodeExtensions.cs]: Why not?

Comment (sharwell) [reply src/Workspaces/Core/Portable/Shared/Extensions/SyntaxNodeExtensions.cs]: It is not related to the subject matter of this pull request, therefore unnecessarily increasing the complexity and time for review, and producing misleading git history results.

**Comment by CyrusNajmabadi** [comment]
I was reviewing all the parent-related helpers.  It for the the purview of that review for me.

**Comment by CyrusNajmabadi** [comment]
Feel free to merge.  thanks! :)

**Comment by CyrusNajmabadi** [comment]
Thanks!
